mods/cnc/mod.yaml

```diff
@@ -209,6 +209,10 @@ SpriteSequenceFormat: TilesetSpecificSpriteSequence
 ModelSequenceFormat: PlaceholderModelSequence
 
 GameSpeeds:
+	slowest:
+		Name: Slowest
+		Timestep: 80
+		OrderLatency: 3
```

Comment:
I would make this value 2 here, it'd still be longer than Slower's time window (160 ms vs 150 ms).

Reply:
I agree.

Reply:
@reaperrr  & @GraionDilach  - Thx for feedback. Updated latency is applied here.